src/Credentials.js

```diff
@@ -176,7 +177,12 @@ class Credentials {
     if (params.verified) payload.verified = params.verified
     if (params.notifications) payload.permissions = ['notifications']
     if (params.callbackUrl) payload.callback = params.callbackUrl
-    if (params.network_id) payload.net = params.network_id
+    if (params.networkId) payload.net = params.networkId
+    if (params.rpcUrl) {
+      if (params.networkId) {
+        payload.rpc = params.rpcUrl
+      } else return Promise.reject(new Error(`rpcUrl was specified but no network_id`))
```

Comment:
Kind of confusing to mix syntax of braces / no-braces with this if statement.  Suggest changing one of these to be consistent with the other.